version/version.h

```diff
@@ -0,0 +1 @@
+#define VERSION (0x00000001)
```

Comment:
It looks like you are using hex value.
We do not support hex versioning, please confirm you are using BCD or correct this.